src/util/jpeg_resolution.cc

```diff
@@ -76,7 +76,7 @@ static int Get16m(const void* Short)
 
 static int ioh_fgetc(const std::unique_ptr<IOHandler>& ioh)
 {
-    uchar c[1] = { 0 };
+    uchar c[1] = {};
```

Comment:
This sets the first entry to 0. I'd keep that for clarity.

Reply:
It sets both to 0. Changed anyway.